### PR TITLE
lib/stdio: Handle 64bits off_t correctly

### PR DESCRIPTION
**Pull request by xiaoxiang781216**
## Impact
Minor

## Testing
Pass CI and ostest

### CODE REVIEW

File: libs/libc/stdio/lib_ftell.c [hunk 1]
```diff
@@ -22,101 +22,13 @@
...
- *
- * Description:
- *   ftell() returns the current value of the file position indicator for the
- *   stream pointed to by stream.
- *
- * Returned Value:
- *   Zero on success; -1 on failure with errno set appropriately.
- *
- ****************************************************************************/
-
```
Comment (hartmannathan) [inline libs/libc/stdio/lib_ftell.c:96]: Why eliminate this docstring?

File: libs/libc/stdio/lib_fseeko.c [hunk 2]
```diff
@@ -22,13 +22,66 @@
...
+/****************************************************************************
+ * Name: fseeko
+ *
+ * Description:
+ *   The fseeko() function sets the file position indicator for the stream
+ *   pointed to by stream. The new position, measured in bytes, is obtained
+ *   by adding offset bytes to the position specified by whence. If whence is
+ *   set to SEEK_SET, SEEK_CUR, or SEEK_END, the offset is relative to the
+ *   start of the file, the current position indicator, or end-of-file,
+ *   respectively. A successful call to the fseek() function clears the
```
Comment (hartmannathan) [inline libs/libc/stdio/lib_fseeko.c:48]: Should be "A successful call to the fseeko() function" -- fseek() -> fseeko()

File: libs/libc/stdio/lib_fseek.c [hunk 3]
```diff
@@ -22,66 +22,13 @@
...
- *   start of the file, the current position indicator, or end-of-file,
- *   respectively. A successful call to the fseek() function clears the
- *   end-of-file indicator for the stream and undoes any effects of the
- *   ungetc(3) function on the same stream.
- *
- * Returned Value:
- *   Zero on success; -1 on failure with errno set appropriately.
- *
- ****************************************************************************/
-
```
Comment (hartmannathan) [inline libs/libc/stdio/lib_fseek.c:56]: Should really eliminate this docstring?

File: libs/libc/stdio/lib_ftell.c (hunk 1 above)
Comment (xiaoxiang781216) [reply libs/libc/stdio/lib_ftell.c:96]: Restore

File: libs/libc/stdio/lib_fseek.c (hunk 3 above)
Comment (xiaoxiang781216) [reply libs/libc/stdio/lib_fseek.c:56]: Restore

**Review by hartmannathan — APPROVED** [review]
LGTM thanks!

**Comment by xiaoxiang781216** [comment]
@hartmannathan please merge this PR if there isn't more issue.

**Comment by hartmannathan** [comment]
Done! Thanks

**Pull request merged**